xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererDXVA.cpp

```diff
@@ -49,18 +49,11 @@ void CRendererDXVA::GetWeight(std::map<RenderMethod, int>& weights, const VideoP
 
   if (av_pixel_format == AV_PIX_FMT_D3D11VA_VLD)
   {
-    // Check if HDR10 passthrough is supported by DXVA video processor
-    // Also used for HLG because it is not supported by Windows, HDR10 is used instead
-    if (streamIsHDR && systemUsesHDR && !enumerator.IsPQ10PassthroughSupported())
```

Comment:
Unused variable 😛

`kodi\xbmc\cores\VideoPlayer\VideoRenderers\windows\RendererDXVA.cpp(38,14): warning C4189: 'streamIsHDR': local variable is initialized but not referenced`

Reply:
indeed